source/aggregation-pipeline-builder.txt

```diff
@@ -14,65 +14,59 @@ Aggregation Pipeline Builder
 
 *New in version 1.14.0*
 
-The Aggregation Pipeline Builder in |compass| provides the ability to
-create :manual:`aggregation pipelines </core/aggregation-pipeline/>` to
-process data. With aggregation pipelines, documents in a collection
-or view pass through multiple stages where they are processed into a
-set of aggregated results. The particular stages and results can be
-modified depending on your application's needs.
+Create :manual:`aggregation pipelines </core/aggregation-pipeline/>` to
+process data. In aggregation pipelines, documents in a collection
+or view pass through stages where |compass| processes them into a
+set of aggregated results. You can change the stages and results to
+suit your needs.
 
-To access the aggregation pipeline builder, navigate to the
-:doc:`collection or view </collections>` for which you wish to create an
-aggregation pipeline and click the :guilabel:`Aggregations` tab. You
-are presented with a blank aggregation pipeline. The
-:guilabel:`Preview of Documents in the Collection` section of the
-:guilabel:`Aggregations` view displays 20 documents sampled from the
-current collection.
+To start building an aggregation pipeline for a :doc:`collection or view
+</collections>`, choose the collection and click the
+:guilabel:`Aggregations` tab.
+A blank aggregation pipeline displays. The :guilabel:`Preview of
+Documents in the Collection` section of the :guilabel:`Aggregations`
+view displays 20 documents sampled from the current collection.
```

Comment:
The 20 documents we show is a configurable number. Maybe we can show that somehow:
`view displays 20 (configurable amount) documents sampled`

I'm not sure if it's just for me, but I think the previous wording for the blank aggregation pipeline read a little more smoothly:
`You are presented with a blank aggregation pipeline.`
vs
`A blank aggregation pipeline displays.`

Maybe we can say `A blank aggregation pipeline is displayed.`
All fine by my me though, just a couple ideas.